openshift/cloud-deploy/deploy.sh

```diff
@@ -3,7 +3,9 @@
 # We use $PTH_ENV in ../deploy.sh
 source conf
 
-${CLOUD_DEPLOYER_PATH}/deploy.sh --conf conf --services services --secrets-template secrets-template-aws.yaml --secrets-deploy ../secrets-deploy.sh --system-deploy ../deploy.sh
-
+# We need to deploy these secrets first
 curl -O https://gitlab.cee.redhat.com/bayesian/secrets/raw/master/secrets-template.yaml
 ../secrets-deploy.sh --secrets-file secrets-template.yaml
+
+${CLOUD_DEPLOYER_PATH}/deploy.sh --conf conf --services services --secrets-template secrets-template-aws.yaml --secrets-deploy ../secrets-deploy.sh --system-deploy ../deploy.sh
```

Comment:
Well, actually the CLOUD_DEPLOYER_PATH should point to a different repo, which actually does the deployment of AWS resources as well.